src/components/VersionChooser/index.tsx

```diff
@@ -36,6 +40,22 @@ type Props = PublicProps &
   PropsFromState &
   RouterProps;
 
+export const higherVersionsThan = (versionId: string) => {
+  if (!versionId) {
+    return Boolean;
+  }
+
+  return (version: VersionsListItem) => version.id > parseInt(versionId, 10);
+};
+
+export const lowerVersionsThan = (versionId: string) => {
+  if (!versionId) {
+    return Boolean;
```

Comment:
This is only needed for Storybook (otherwise we won't have any value). I don't know if we want that, but on the other hand, it won't hurt.